strawberryfields/program.py

```diff
@@ -433,6 +433,60 @@ def append(self, op, reg):
         self.circuit.append(Command(op, reg))
         return reg
 
+    def _is_select(self, op):
+        """Check if the quantum operation uses post-selection.
+
+        A quantum operation uses post-selection if it has the attribute `select`
+        and this attribute is not None.
+
+        Args:
+            op (Operation): quantum operation
+        Returns
+            (bool): boolean indicating if the quantum operation uses post-selection
+                or not
+        """
+
+        return hasattr(op, "select") and getattr(op, "select") is not None
+
+    def _is_feed_forward(self, op):
+        """Check if the quantum operation uses feed-forwarding.
+
+        A quantum operation uses feed-forwarding if its attribute
+        `measurement_deps` is a non empty set.
+
+        Args:
+            op (Operation): quantum operation
+        Returns
+            (bool): boolean indicating if the quantum operation uses feed-forwarding
+                or not
+        """
+
+        return bool(op.measurement_deps)
+
+    @property
+    def post_selection(self):
```

Comment:
Maybe change this name to something more descriptive. E.g., `has_post_selection`? 🤔

Reply:
Sure, I just used the same _names_ proposed in the issue #302. I'm going to make the changes :+1: